lib/assembly-objectfile.rb

```diff
@@ -25,8 +25,9 @@ module Assembly
   FILE_ATTRIBUTES['application/pdf'] = { preserve: 'yes', shelve: 'yes', publish: 'yes' }
   FILE_ATTRIBUTES['plain/text'] = { preserve: 'yes', shelve: 'yes', publish: 'yes' }
   FILE_ATTRIBUTES['text/plain'] = { preserve: 'yes', shelve: 'yes', publish: 'yes' }
-  FILE_ATTRIBUTES['image/png'] = { preserve: 'no', shelve: 'yes', publish: 'yes' }
+  FILE_ATTRIBUTES['image/png'] = { preserve: 'yes', shelve: 'yes', publish: 'no'}
```

Comment:
@blalbrit do you know if this change makes sense?  it doesn't seem related to 3d, so not sure why I made it?

Reply:
That change was made, I think, because they didn't want the png files listed in the public content metadata (the files are referenced from the 3d model, but the embed viewer shouldn't try to load them as independent assets.

Reply:
ok, will leave it in the PR